crates/burn-import/src/onnx/op_configuration.rs

```diff
@@ -768,9 +768,20 @@ pub fn tile_config(node: &Node) -> TileConfig {
 
 /// Create a PadConfig from the attributes of the node
 pub fn pad_config(node: &Node) -> PadConfig {
+    fn get_input_values(node: &Node, index: usize) -> Vec<i64> {
+        // If the input is not provided, return an empty vector
+        if node.inputs.get(index).is_none() {
+            return Vec::new();
+        }
+
+        match &node.inputs[index].value {
+            Some(Data::Int64s(shape)) => shape.clone(),
+            _ => panic!("Tensor data type must be int64"),
+        }
+    }
```

Comment:
Did you intend to include pad related changes? I see you have #2195 open. Perhaps, this is based on that branch and that's why these changes appear?

Reply:
Yeah, I had to make the pad related changes in order to test the user model.